libs/@guardian/source-react-components/src/button/theme.ts

```diff
@@ -36,7 +38,8 @@ export const buttonThemeDefault: { button: ButtonTheme } = {
 		textSecondary: palette.brand[400],
 		backgroundSecondary: palette.brand[800],
 		backgroundSecondaryHover: '#ACC9F7', // One-off colour variant generated from palette.brand[800]
-		textTertiary: palette.brand[400],
+		textTertiary: palette.neutral[100],
+		backgroundTertiary: palette.neutral[100],
```

Comment:
Do we need to be updating this deprecated object.

Reply:
Not sure… it felt safer but will follow your lead!